src/status_im/transport/message/core.cljs

```diff
@@ -75,6 +76,15 @@
                   (activity-center/show-toasts notifications)
                   (process-next response-js sync-handler)))
 
+      (some? activity-center-state)
```

Comment:
This piece of code looks consistent with the existing code in the function.

As a side question not related to your work:
Are we planning on refactor this implementation to use a functional-clojure style? I mean, avoiding those `js-delete` and extract keys from a Clojure map instead of a JS Object (response-js).

Reply:
That's actually a very good question @ulisesmac. These functions should/could be refactored, we just need to make sure we don't make them any slower after refactoring.

I remember @cammellos once told me that we (in this mobile universe) don't have the luxury of converting all data structures from JS to CLJS because it's rather expensive. Unfortunately some performance characteristics we take for granted in web development are significantly affected in some mobile devices.

> I mean, avoiding those js-delete and extract keys

Yeah, and while developing I actually lost almost 1h of my time because I forgot to call `js-delete`, otherwise I got an infinite loop.

Reply:
Yes, this is the part of the codebase where we want to be a bit careful with performance, so we don't go straight for clojure structs, if I recall correctly, converting js->clj was taking roughly 8ms of the 16ms that we should have available to stay within the 60fps

Reply:
Oh wow! What a big amount of time!

Have you tried Transit as suggested by the docs? If so, does it help?

 https://cljs.github.io/api/cljs.core/js-GTclj

> Note that js->clj is not optimized for speed and the transit.cljs library is recommended for parsing large amounts of JSON data.

Reply:
We used transit before (actually the whole network protocol used to be in transit, now we use protobufs), and its performance were not great either, we moved away from it completely, that was around 2/3 years ago though, maybe things have changed, but raw javascript is generally much faster

Reply:
Ok! Thanks for clarifying 👍